### PR TITLE
Fill in MALAMUTE SQA documents

### PR DESCRIPTION
**Pull request by cticenhour**
Refs #73

### CODE REVIEW

File: doc/content/sqa/malamute_cci.md [hunk 1]
```diff
@@ -1 +1,13 @@
-!template load file=sqa/app_cci.md.template app=MalamuteApp category=malamute
\ No newline at end of file
+!template load file=sqa/app_cci.md.template app=MALAMUTE category=malamute
+
+!template item key=general-communication
+Please use the [{{app}} Discussion forum](https://github.com/idaholab/{{category}}/discussions) for all
+questions and inquires.
```
Comment (lindsayad) [inline doc/content/sqa/malamute_cci.md]:
```suggestion
questions and inquiries.
```

File: doc/content/sqa/malamute_scs.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+testing system, [CIVET](https://civet.inl.gov), that is triggered when
+you submit a pull request will check your code and provide information on the
+changes needed to conform to the code style (if any).
+
+## MOOSE Code Standard
+
+!style halign=left
+For extended information on file guidelines, naming conventions, example code,
+doxygen documentation, and other tips, please see
+[the MOOSE Code Standards](sqa/framework_scs.md).
```
Comment (lindsayad) [inline doc/content/sqa/malamute_scs.md]: Is MOOSE correct here or should it be MALAMUTE? Maybe the header title should be MALAMUTE and then we say something about how it follows MOOSE code standards?

File: doc/content/sqa/malamute_srs.md [hunk 3]
```diff
@@ -1 +1,65 @@
...
+!template! item key=system-scope
+!! system-scope-begin
+
+MALAMUTE performs simulations related to [!ac](AM) processes. These models often include highly coupled
+systems of equations related to heat conduction, electromagnetics, Navier Stokes, mechanics, and surface
+morphology, amongst others. Material models are also included to support these simulations (such as
+graphite and stainless steel), and they themselves are often dependent on simulation variables: temperature,
+electromagnetic field strength, stress/strain, etc. While many models within MALAMUTE are performed
+at the "engineering scale" (i.e., at the scale of centimeters and meters), the [syntax/MultiApps/index.md]
+can leveraged to allow for multiscale coupling to the micro- and nano-scale of a given experiment.
```
Comment (lindsayad) [inline doc/content/sqa/malamute_srs.md]:
```suggestion
can be leveraged to allow for multiscale coupling to the micro- and nano-scale of a given experiment.
```

**Review by lindsayad — CHANGES_REQUESTED** [review]

File: doc/content/sqa/malamute_scs.md (hunk 2 above)
Comment (cticenhour) [reply doc/content/sqa/malamute_scs.md]: Good idea - will adjust.

Comment (cticenhour) [inline doc/content/sqa/malamute_scs.md]:
```suggestion
## MALAMUTE Code Standards

!style halign=left
MALAMUTE follows the MOOSE code standards for all development. For information on file guidelines, naming conventions, example code,
doxygen documentation, and other tips, please see
[the MOOSE standard here](sqa/framework_scs.md).
```

**Review by lindsayad — APPROVED** [review]

**Comment by cticenhour** [comment]
Thanks!

**Pull request merged**